src/averageAges.js

```diff
@@ -37,7 +53,15 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const onlyWomen = people.filter(person => {
+    const Woman = person.sex === 'f';
```

Comment:
we use capital letter for classes or constructors
```suggestion
    const women = person.sex === 'f';
```